test/dice.test.ts

```diff
@@ -108,7 +107,7 @@ describe ("Dice", () => {
     });
 
     it ("should __NOT__ detect 'a Three of a kind' before 'Full House' in [2, 5, 2, 5, 2]", () => {
-      // Full House takes precedence ofer Three of a Kind, so we have to
+      // Full House takes precedence offer Three of a Kind, so we have to
```

Comment:
👌 💯 🔥

Reply:
Should be `over` 🏃 💨

Reply:
This is the highlight of the PR

Reply:
resolved by commit ff1e8acb518c6081ddef5303fd6ec02575d43194